src/lib/hooks/useEnableKeyboardController.tsx

```diff
@@ -1,108 +0,0 @@
-import {
```

Comment:
Currently, RNKC couldn't be disable (at least not until the fix lands).

That's why this PR embrace it totally: instead of a mix of RNKC / the builtin `KeyboardAvoidingView` and the default basic Android handling (with `android:configChanges="keyboard|keyboardHidden`).

Totally switching to RNKC can bring consistency between iOS and Android, and is the safest option to prepare for Android 15 (SDK 35 - with enforced edge-to-edge) but require a bit of work.